media-gfx/zbar/files/zbar-0.23-fix_QX11Info.patch

```diff
@@ -0,0 +1,12 @@
+diff -ur a/qt/QZBar.cpp b/qt/QZBar.cpp
+--- a/qt/QZBar.cpp	2019-05-22 14:00:30.000000000 +0300
++++ b/qt/QZBar.cpp	2020-01-06 09:40:47.344538896 +0300
+@@ -23,7 +23,7 @@
+ 
+ #include <qevent.h>
+ #include <qurl.h>
+-#include <QX11Info>
++#include <QtX11Extras/QX11Info>
```

Comment:
why?

Reply:
because without this patch using the `qt5` flag I get :
```bash
make[2]: Entering directory '/tmp/portage/media-gfx/zbar-0.23/work/zbar-0.23-abi_x86_64.amd64'
/bin/sh ./libtool  --tag=CXX   --mode=compile x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I. -I/tmp/portage/media-gfx/zbar-0.23/work/zbar-0.23 -I./include  -Iqt -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I/usr/include/qt5/QtWidgets -I/usr/include/qt5 -I/usr/include/qt5/QtGui -I/usr/include/qt5 -I/usr/include/qt5/QtCore -I/usr/include/qt5 -I/tmp/portage/media-gfx/zbar-0.23/work/zbar-0.23/include -DNDEBUG  -fPIC -Wall -Wno-parentheses -march=x86-64 -O2 -pipe -c -o qt/libzbarqt_la-QZBar.lo `test -f 'qt/QZBar.cpp' || echo '/tmp/portage/media-gfx/zbar-0.23/work/zbar-0.23/'`qt/QZBar.cpp
libtool: compile:  x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I. -I/tmp/portage/media-gfx/zbar-0.23/work/zbar-0.23 -I./include -Iqt -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I/usr/include/qt5/QtWidgets -I/usr/include/qt5 -I/usr/include/qt5/QtGui -I/usr/include/qt5 -I/usr/include/qt5/QtCore -I/usr/include/qt5 -I/tmp/portage/media-gfx/zbar-0.23/work/zbar-0.23/include -DNDEBUG -fPIC -Wall -Wno-parentheses -march=x86-64 -O2 -pipe -c /tmp/portage/media-gfx/zbar-0.23/work/zbar-0.23/qt/QZBar.cpp  -fPIC -DPIC -o qt/.libs/libzbarqt_la-QZBar.o
/tmp/portage/media-gfx/zbar-0.23/work/zbar-0.23/qt/QZBar.cpp:26:10: fatal error: QX11Info: No such file or directory
   26 | #include <QX11Info>
      |          ^~~~~~~~~~
compilation terminated.
make[2]: *** [Makefile:1630: qt/libzbarqt_la-QZBar.lo] Error 1
make[2]: Leaving directory '/tmp/portage/media-gfx/zbar-0.23/work/zbar-0.23-abi_x86_64.amd64'
make[1]: *** [Makefile:1873: all-recursive] Error 1
make[1]: Leaving directory '/tmp/portage/media-gfx/zbar-0.23/work/zbar-0.23-abi_x86_64.amd64'
make: *** [Makefile:953: all] Error 2
 * ERROR: media-gfx/zbar-0.23::gentoo failed (compile phase):
 *   emake failed
```

Reply:
maybe I need to try to get around this problem somehow differently?